src/styles/blocks/icon-bar.scss

```diff
@@ -0,0 +1,44 @@
+.icon-bar {
+  display: flex;
+  justify-content: flex-end;
+  flex-basis: 100%;
+
+  &__item {
+    border-left: 0.5px solid $separator-color;
+  }
+
+  &__link {
+    position: relative;
+    display: flex;
+    transition: background-color $transition-duration;
+
+    &:hover {
+      background-color: $bg-controllers;
+    }
+
+    &--active {
+      color: $theme-color-light;
+
+      &::after {
+        position: absolute;
+        bottom: -2px;
+        content: "";
+        height: 3px;
+        background: $theme-color-light;
+        border-radius: 4px;
+
+        animation: underline $transition-duration ease forwards;
+      }
+    }
+  }
+
+  &__icon {
+    margin: 16px;
+    height: 16px;
+    fill: $theme-color-light;
+
+    @include onTablet {
+      margin: 24px;
+    }
+  }
+}
```

Comment:
Put components into an src/components folder.
each component should be a folder with index.ts, ComponentName.tsx, ComponentName.module.scss files.
use CSS modules.
keep .module.scss files together with their components.